### PR TITLE
Build system cleanup

### PR DESCRIPTION
**Pull request by cwendling**
Followups and fixes for #5 and general build system stuff.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+po/*.sed
+po/*.sin
+po/*.header
+po/Rules-quot
+po/Makevars.template
+po/Makefile.in.in
+po/POTFILES
+po/mate-wayland-session.pot
+po/stamp-po
+session/MATE.desktop
```
Comment (cwendling) [inline .gitignore:26]: Mybe we want to use [git.mk](https://raw.githubusercontent.com/behdad/git.mk/) here as we do in other repos, in which case the commit adding this `.gitignore` can be removed from this PR.

Comment (lukefromdc) [reply .gitignore:26]: I have zero familiarity with using that, will go with .gitignore for the moment but look at that for potential later use

**Comment by lukefromdc** [comment]
Merged with
https://github.com/mate-desktop/mate-wayland-session/commit/07c31ea5776c61bd142762a49e3068d1ac04264c
including use of git.mk and MAINTAINERCLEANFILES listings for those files other packages wouild remove on running make maintainer-clean

**Comment by lukefromdc** [comment]
Command line merge necessary to handle additional commits for git.mk, removal of manual .gitignore commit, and squash without write access to the branch used for the PR. Thanks, this gives much cleaner tarballs!

**Comment by lukefromdc** [comment]
Next problem is how to send a tarball to https://pub.mate-desktop.org/releases/1.28/
given I do not have Travis working on this, and an attempt to trigger a build after adding the two .yml files used netted a "too many users for account" error at Travis.

**Comment by cwendling** [comment]
@lukefromdc where do you see this error?  I just see the repository with no info linked on https://app.travis-ci.com/github/mate-desktop/mate-wayland-session but I don't see any detail there.

**Comment by cwendling** [comment]
@lukefromdc wait, shouldn't you simply `mv travis.yml .travis.yml`?  Or give me push rights so I can try stuff :)

**Comment by lukefromdc** [comment]
OK, I found that when I created this repo, github didn't give the whole team access. I added you and core-team to deal with this for now.,

**Comment by lukefromdc** [comment]
> @lukefromdc where do you see this error? I just see the repository with no info linked on https://app.travis-ci.com/github/mate-desktop/mate-wayland-session but I don't see any detail there.

Lost track of that one, concerns only manually triggering a build.

**Pull request closed without merging**